### PR TITLE
setting node version on travis

### PR DESCRIPTION
**Pull request by mamute**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,8 @@ rvm:
   - 2.2.3
 install:
   - bundle install
+  - nvm install 5.0
+  - nvm use 5.0
   - npm install -g npm
```
Comment (dylangrafmyre) [inline .travis.yml]: @justin808 @mamute Since we are now install Node 5.0 with nvm, we can probably remove `npm install -g npm`. Node 5.0 comes with NPM 3.x.x.

Before travis was using a older version of npm that was failing our build.

Comment (mamute) [inline .travis.yml]: done :+1:

Comment (justin808) [inline .travis.yml]: Hi @dylangrafmyre I just merged. Let's do another PR if that's the case.

BTW, I get the linting error locally, so this is probably not the main build issue. There's some problem with eslint?

Comment (dylangrafmyre) [inline .travis.yml]: @justin808 the main build failure is from a c++ compiler error for `NAN` module used in sleep. I will fix this.
